packages/lab/src/__tests__/__e2e__/file-drop-zone/FileDropZone.cy.tsx

```diff
@@ -0,0 +1,80 @@
+import { composeStories } from "@storybook/react";
+import * as fileDropZoneStories from "@stories/file-drop-zone/file-drop-zone.stories";
+import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessibility";
```

Comment:
not really related but I am generally wondering why checkAccessibility is in a completely different location when it's only imported in files in this location

Reply:
probably because the cypress folder would be a good place to add general tests, also they get used in core and lab, but @joshwooding might have a much better sounding explanation :)